src/platforms/dotnet/guides/winui/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: WinUI 3
+description: "Learn about using Sentry's .NET SDK with the Windows UI Library (WinUI 3)"
+redirect_from:
+  - /platforms/dotnet/winui3/
+---
+
+Sentry's .NET SDK works with Windows UI Library applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry).
+
+## Configuration (for Just-In-Time compilation)
+
+If you are using Just-In-Time (JIT) compilation, which will be the case for most users, all you need to do is [initialize the SDK with `SentrySdk.Init`](/platforms/dotnet/) in the constructor of your application class (usually `App.xaml.cs`).
```

Comment:
> which will be the case for most users

Is this true? UWP is only AOT. Perhaps what they say about "Desktop" means just .NET (JIT). So maybe we need to reword here to be specific when it's going to be JIT and when it's AOT (particularly now that even .NET has AOT as first-class with NativeAOT).